applications/StructuralMechanicsApplication/python_scripts/RVEAnalysis.py

```diff
@@ -0,0 +1,353 @@
+# makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
this file should be names `rve_analysis.py`

Reply:
ok